pageserver/src/tenant/layer_map/historic_layer_coverage.rs

```diff
@@ -80,11 +85,11 @@ impl<Value: Clone> HistoricLayerCoverage<Value> {
         }
     }
 
-    /// Add a layer
+    /// Add a layer, returning whether the operation was a noop.
```

Comment:
Consider returning 'false' rather than 'true' on noop. Then 'true' would mean "did something". It would avoid double negatives, as "noop" already means "didn't do anything".